Bio/AlignIO/StockholmIO.py

```diff
@@ -456,7 +456,16 @@ def __next__(self):
                 elif line[:5] == "#=GS ":
                     # Generic per-Sequence annotation, free text
                     # Format: "#=GS <seqname> <feature> <free text>"
-                    seq_id, feature, text = line[5:].strip().split(None, 2)
+                    gsannotation = line[5:].strip().split(None, 2)
+                    print(gsannotation)
+                    seq_id = gsannotation[0]
+                    feature = gsannotation[1]
+                    # Free text can sometimes be empty, which a one line split throws an error for.
+                    # See https://github.com/biopython/biopython/issues/2982 for more details
+                    if len(gsannotation) == 3:
+                        text = gsannotation[2]
+                    elif len(gsannotation) == 2:
+                        text = ""
```

Comment:
How about something like this:

```python
try:
    seq_id, feature, text = line[5:].strip().split(None, 2)
except ValueError:
    # Free text can sometimes be missing
    seq_id, feature = line[5:].strip().split(None, 1)
    text = ""
```

At very least, you need to remove the print line.

Reply:
Great improvement, thanks! The length way of doing it was not ideal. I can't imagine there would be any risk in a `ValueError` at this point, can you?

Apologies for the print line. I am still getting used to the biopython tests and must have done `git reset --soft HEAD` a couple of times too many!

I am following [CONTRIBUTING.rst](https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst) for contributing. Is that the most up to date guide?

Reply:
The CONTRIBUTING file is up to date, but perhaps could be extended further. As a new contributor you may pickup on things we're just assuming without writing down?

It could point at https://biopython.org/wiki/GitUsage but that is a little out of date (we've been using git for years now). e.g. We like to keep a simple linear history so pull requests are either done with a rebase-and-merge, or a squash-and-merge